.github/mergify.yml

```diff
@@ -0,0 +1,42 @@
+pull_request_rules:
+- name: auto-merge
+  description: automatic merge for main with > 1 approved reviews, all requested reviews have given feedback, not held, and CI is successful
+  conditions:
+    - "#approved-reviews-by>=1"
+    - "#review-requested=0"
```

Comment:
You're right. This would depend on dropping `CODEOWNERS` that's automatically requesting reviews from a large group on every PR -- https://github.com/instructlab/instructlab/pull/938

Reply:
Is this just for _anyone_ approving? Because it should only be an approval from the CLI Maintainers per the merge policy https://github.com/instructlab/enhancements/blob/main/docs/github-merge-strategy.md

Reply:
Note that #938 is merged, so this condition should now be OK.